app/packs/review/page.tsx

```diff
@@ -12,24 +13,20 @@ export const metadata: Metadata = {
 async function Review() {
 	const packs = await prisma.questionPack.findMany({
 		where: {
-			OR: [{ pending: true }, { denied: true }]
+			OR: [{ status: Status.pending }, { status: Status.pending }]
 		},
 		select: {
 			type: true,
 			id: true,
 			name: true,
-			pending: true,
 			description: true,
-			denied: true,
-			questions: true
+			questions: true,
+			status: true // Include status to handle packs directly
 		}
-		// Maybe paginate?
 	})
 
-	// Split results
-	const denied = packs.filter((pack) => pack.denied)
-	const pending = packs.filter((pack) => pack.pending)
-
+	// Filtered directly by status, no need to split
+	const pending = packs.filter((pack) => pack.status === Status.pending)
```

Comment:
This should be replaced with a sort from pending -> resubmit_pending
```suggestion
	const pending = packs.sort((a, b) => {
		if (a.status === Status.pending) return -1
		if (b.status === Status.pending) return 1
		if (a.status === Status.resubmit_pending) return -1
		if (b.status === Status.resubmit_pending) return 1
		return 0
	})
```